pom.xml

```diff
@@ -66,9 +66,9 @@
     </pluginRepositories>
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.387.3</jenkins.version>
-        <!-- TODO until in plugin-pom -->
-        <jenkins-test-harness.version>2042.v787a_641a_9b_26</jenkins-test-harness.version>
+        <!-- TODO Until in plugin-pom -->
```

Comment:
https://github.com/jenkinsci/plugin-pom/pull/908 needs release apparently. Maybe we should switch to CD.